realm/realm-library/src/main/java/io/realm/BaseRealm.java

```diff
@@ -45,7 +45,7 @@
  * @see io.realm.Realm
  * @see io.realm.DynamicRealm
  */
-abstract class BaseRealm implements Closeable {
+public abstract class BaseRealm implements Closeable {
```

Comment:
It is an unfortunate side-effect of having to make `getRealm` part of the interface, which is needed for weak change listeners in RealmBaseAdapter :(